lms/djangoapps/mobile_api/course_info/tests.py

```diff
@@ -84,60 +84,95 @@ def test_updates(self, new_format):
+            response = self.api_response(expected_response_code=200)
+            self.assertIsNone(response.data['handouts_html'])
+
+    @ddt.data(ModuleStoreEnum.Type.mongo, ModuleStoreEnum.Type.split)
+    def test_handouts_static_rewrites(self, default_ms):
+        with self.store.default_store(default_ms):
+            self.add_mobile_available_toy_course()
+
+            # check that we start with relative static assets
+            handouts_usage_key = self.course.id.make_usage_key('course_info', 'handouts')
+            underlying_handouts = self.store.get_item(handouts_usage_key)
+            self.assertIn('\'/static/', underlying_handouts.data)
+
+            # but shouldn't finish with any
+            response = self.api_response()
+            self.assertNotIn('\'/static/', response.data['handouts_html'])
+
+    @ddt.data(ModuleStoreEnum.Type.mongo, ModuleStoreEnum.Type.split)
+    def test_jump_to_id_handout_href(self, default_ms):
+        with self.store.default_store(default_ms):
+            self.add_mobile_available_toy_course()
+
+            # check that we start with relative static assets
+            handouts_usage_key = self.course.id.make_usage_key('course_info', 'handouts')
+            underlying_handouts = self.store.get_item(handouts_usage_key)
+            underlying_handouts.data = "<a href=\"/jump_to_id/identifier\">Intracourse Link</a>"
+            self.store.update_item(underlying_handouts, self.user.id)
+
+            # but shouldn't finish with any
+            response = self.api_response()
+            self.assertIn("/courses/{}/jump_to_id/".format(self.course.id), response.data['handouts_html'])
+
+    @ddt.data(ModuleStoreEnum.Type.mongo, ModuleStoreEnum.Type.split)
+    def test_course_url_handout_href(self, default_ms):
+        with self.store.default_store(default_ms):
+            self.add_mobile_available_toy_course()
+
+            # check that we start with relative static assets
+            handouts_usage_key = self.course.id.make_usage_key('course_info', 'handouts')
+            underlying_handouts = self.store.get_item(handouts_usage_key)
+            underlying_handouts.data = "<a href=\"/course/identifier\">Linked Content</a>"
+            self.store.update_item(underlying_handouts, self.user.id)
+
+            # but shouldn't finish with any
+            response = self.api_response()
+            self.assertIn("/courses/{}/".format(self.course.id), response.data['handouts_html'])
+
+    def add_mobile_available_toy_course(self):
+        """ use toy course with handouts, and make it mobile_available """
+        course_items = import_course_from_xml(self.store, self.user.id, settings.COMMON_TEST_DATA_ROOT, ['toy'],
+                                              create_if_not_present=True)
         self.course = course_items[0]
         self.course.mobile_available = True
         self.store.update_item(self.course, self.user.id)
-
-    def verify_success(self, response):
```

Comment:
Why was the implementation of `verify_success` removed?  It overrides the `verify_success` method in the `Mixin`.